src/main/client/src/components/@commons/Warning.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react'
+import { Alert, AlertDescription, AlertIcon, AlertTitle, CloseButton } from '@chakra-ui/react'
+import axios from 'axios'
+import { API_BASE_URL } from '../../utils/configurations'
+import { WarningView } from 'types/dto/warning'
+import { Container } from 'components/@layout/Container'
+
+export const Warning: React.FC = () => {
+  const [warning, setWarning] = React.useState<WarningView | undefined>(undefined)
+  const [closed, setClosed] = React.useState<boolean>(false)
+
+  const closeWarning = () => setClosed(true)
+
+  React.useEffect(() => {
+    axios.get<WarningView>(`${API_BASE_URL}/api/warning`).then((res) => {
+      if (res.data.message !== '') setWarning(res.data)
+    })
+  }, [setWarning])
+
+  return warning === undefined || closed ? (
+    <></>
```

Comment:
`(warning || closed) && <Component/>`. Also use null instead of empty tag in the future.

Reply:
Well more like:

```tsx
return (warning === undefined || closed) && (
  <Container>
     <Alert>
       ...
     </Alert>
   </Container>
)
```

Reply:
`if(warning===undefined || closed) return null`

Reply:
Lol I'm dumb, so there should be a `!` at the start xd